flow360/component/flow360_params/flow360_output.py

```diff
@@ -0,0 +1,323 @@
+"""
+Flow360 output parameters models
+"""
+from abc import ABC
+from typing import List, Literal, Optional, Union, get_args
+
+import pydantic as pd
+
+from ..types import Coordinate, PositiveInt
+from .flow360_fields import (
+    CommonFieldVars,
+    IsoSurfaceFieldVars,
+    SurfaceFieldVars,
+    VolumeSliceFieldVars,
+)
+from .params_base import (
+    Flow360BaseModel,
+    Flow360SortableBaseModel,
+    _self_named_property_validator,
+)
+
+
+class OutputBase(ABC, Flow360BaseModel):
+    """:class: Base class for common output parameters"""
+
+    Cp: Optional[bool] = pd.Field()
+    grad_w: Optional[bool] = pd.Field(alias="gradW")
+    k_omega: Optional[bool] = pd.Field(alias="kOmega")
+    Mach: Optional[bool] = pd.Field(alias="Mach")
+    mut: Optional[bool] = pd.Field()
+    mut_ratio: Optional[bool] = pd.Field(alias="mutRatio")
+    nu_hat: Optional[bool] = pd.Field(alias="nuHat")
+    primitive_vars: Optional[bool] = pd.Field(alias="primitiveVars")
+    q_criterion: Optional[bool] = pd.Field(alias="qcriterion")
+    residual_navier_stokes: Optional[bool] = pd.Field(alias="residualNavierStokes")
+    residual_transition: Optional[bool] = pd.Field(alias="residualTransition")
+    residual_turbulence: Optional[bool] = pd.Field(alias="residualTurbulence")
+    s: Optional[bool] = pd.Field()
+    solution_navier_stokes: Optional[bool] = pd.Field(alias="solutionNavierStokes")
+    solution_turbulence: Optional[bool] = pd.Field(alias="solutionTurbulence")
+    solution_transition: Optional[bool] = pd.Field(alias="solutionTransition")
+    T: Optional[bool] = pd.Field(alias="T")
+    vorticity: Optional[bool] = pd.Field()
+    wall_distance: Optional[bool] = pd.Field(alias="wallDistance")
+    low_numerical_dissipation_sensor: Optional[bool] = pd.Field(
+        alias="lowNumericalDissipationSensor"
+    )
+    residual_heat_solver: Optional[bool] = pd.Field(alias="residualHeatSolver")
```

Comment:
move all bool options to separate class that will be only used in updater but not be part of allowed configuration

Reply:
Fixed in 05ca5e6